test/integration/full-content/full-content.js

```diff
@@ -16,11 +16,7 @@ import {
 	unstable__bootstrapServerSideBlockDefinitions, // eslint-disable-line camelcase
 } from '@wordpress/blocks';
 import { parse as grammarParse } from '@wordpress/block-serialization-spec-parser';
-
```

Comment:
@youknowriad I think this file no longer runs as part of `npm run test-unit`? Is that intentional? Additionally `require( '../../packages/editor/src/hooks' );` won't work if forced to run.

Reply:
Good catch, It was not intentional, I just moved it to the integration tests folder but apparently this folder needs a `.spec` in the file name to run tests.

Reply:
It should be fixed in https://github.com/WordPress/gutenberg/pull/8923